 python /home/jenkins/agent/workspace/DJD/CD-Deploy/openr-pipeline-int/pipeline/lib/scripts/latest_os_factory.py oat-djd-af107-backend '["paris","north"]'
Traceback (most recent call last):
  File "/home/jenkins/agent/workspace/DJD/CD-Deploy/openr-pipeline-int/pipeline/lib/scripts/notify/generate_excel.py", line 3, in <module>
    from openpyxl import Workbook
ModuleNotFoundError: No module named 'openpyxl'
Traceback (most recent call last):
  File "/home/jenkins/agent/workspace/DJD/CD-Deploy/openr-pipeline-int/pipeline/lib/scripts/latest_os_factory.py", line 201, in <module>
    final_result = main()
  File "/home/jenkins/agent/workspace/DJD/CD-Deploy/openr-pipeline-int/pipeline/lib/scripts/latest_os_factory.py", line 151, in main
    subprocess.run(["sh", NOTIFY_SERVER_MISMATCH, server_name, account, mismatch_output], check=True)
  File "/usr/local/lib/python3.10/subprocess.py", line 524, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['sh', '/home/jenkins/agent/workspace/DJD/CD-Deploy/openr-pipeline-int/pipeline/lib/scripts/../scripts/notify/notify_image_server_mismatch.sh', 'oat-djd-af107-backend', 'DJD_AF107_DEV', 'Server Name: [oat-djd-af107-backend-green-new-eu-fr-paris-1] = Image Name: [ocs_dev_RHEL_9_2025_w08]\nServer Name: [oat-djd-af107-backend-green-new-eu-fr-paris-2] = Image Name: [ocs_dev_RHEL_9_2025_w08]\nServer Name: [oat-djd-af107-backend-blue-new-eu-fr-paris-1] = Image Name: [ocs_dev_RHEL_9_2025_w08]\nServer Name: [oat-djd-af107-backend-blue-new-eu-fr-paris-2] = Image Name: [ocs_dev_RHEL_9_2025_w08]\nServer Name: [oat-djd-af107-backend-blue-new-eu-fr-north-1] = Image Name: [OS_dev_RHEL_9_2024_w51]\nServer Name: [oat-djd-af107-backend-green-new-eu-fr-north-1] = Image Name: [OS_dev_RHEL_9_2024_w51]']' returned non-zero exit status 1.
���� Generating Excel report...
��� ERROR: Excel file generation failed!
